Source/WebKit/NetworkProcess/storage/CacheStorageCache.cpp

```diff
@@ -95,19 +95,19 @@ void CacheStorageCache::open(WebCore::DOMCacheEngine::CacheIdentifierCallback&&
     if (m_isInitialized)
         return callback(WebCore::DOMCacheEngine::CacheIdentifierOperationResult { m_identifier, false });
 
-    m_store->readAllRecordInfos([this, weakThis = WeakPtr { *this }, callback = WTFMove(callback)](auto recordInfos) mutable {
+    m_store->readAllRecordInfos([this, weakThis = WeakPtr { *this }, callback = WTFMove(callback)](auto&& recordInfos) mutable {
         if (!weakThis)
             return callback(makeUnexpected(WebCore::DOMCacheEngine::Error::Internal));
 
         std::sort(recordInfos.begin(), recordInfos.end(), [](auto& a, auto& b) {
             return a.insertionTime < b.insertionTime;
         });
 
-        for (auto& recordInfo : recordInfos) {
+        for (auto&& recordInfo : recordInfos) {
             recordInfo.identifier = nextRecordIdentifier();
             m_records.ensure(computeKeyURL(recordInfo.url), [] {
                 return Vector<CacheStorageRecordInformation> { };
-            }).iterator->value.append(recordInfo);
+            }).iterator->value.append(WTFMove(recordInfo));
```

Comment:
We are fixing a UAM but we are doing more WTFMove.
This one seems fine though.

Reply:
I could split them into two patches if you prefer. Do you want me to split it?